signing/unencrypted_file_system_keystore.js

```diff
@@ -18,8 +17,11 @@ class UnencryptedFileSystemKeyStore {
      * @param {string} keypair 
      */
     async setKey(accountId, keypair) {
-        if (!await promisify(fs.exists)(keyDir)){
```

Comment:
nit: I think it makes sense to `promisify` all used `fs` functions at once after requiring `fs` vs doing it inline